confluent-cloud-rest-produce-schema-registry/README.md

```diff
@@ -0,0 +1,187 @@
+<!-- title: How to produce Avro-formatted Kafka messages to Confluent Cloud via REST API -->
+<!-- description: In this tutorial, learn how to produce Avro-formatted Kafka messages to Confluent Cloud via REST API, with step-by-step instructions and supporting code. -->
+
+# How to produce Avro-formatted Kafka messages to Confluent Cloud via REST API
+
+Confluent Cloud provides a set of [REST APIs](https://docs.confluent.io/cloud/current/api.html) for interacting with cloud resources. The [Produce API](https://docs.confluent.io/cloud/current/kafka-rest/kafka-rest-cc.html#produce-api) allows clients to produce records to a Kafka topic via an HTTP POST request. However, this API currently does not support a built-in way to produce records that use schemas from Schema Registry. Despite this limitation, clients can still produce such records by sending binary data that conforms to the message wire format documented [here](https://docs.confluent.io/platform/current/schema-registry/fundamentals/serdes-develop/index.html#wire-format). This tutorial walks through the necessary steps.
+
+## Prerequisites
+
+* A [Confluent Cloud](https://confluent.cloud/signup) account
+* A Kafka cluster in Confluent Cloud (e.g., following this [quick start step](https://docs.confluent.io/cloud/current/get-started/index.html#section-1-create-a-cluster-and-add-a-topic))
+* The [Confluent CLI](https://docs.confluent.io/confluent-cli/current/install.html) installed on your machine
+* [`jq`](https://jqlang.org/download/) for parsing JSON on the command line
+
+## Create a topic and associated schema
+
+To begin, clone the `confluentinc/tutorials` GitHub repository and navigate to the appropriate directory:
+
+```shell
+git clone git@github.com:confluentinc/tutorials.git
+cd tutorials/confluent-cloud-rest-produce-schema-registry
+```
+
+Log in to Confluent Cloud using the Confluent CLI:
+
+```shell
+confluent login --prompt
+```
+
+Create a Kafka topic named `orders`:
+
+```shell
+confluent kafka topic create orders
```

Comment:
If followed as it is will give info that clusters prop is missing --  Error: no Kafka cluster selected

Suggestions:
    You must pass --cluster with the command or set an active Kafka cluster in your context with confluent kafka cluster use.

Reply:
Good catch - I adjusted the instructions to set the active environment / cluster as an initial step